### ISSUE TITLE
Unable to jspm bundle aurelia-dependency-injection 1.4

### ISSUE THREAD

**Issue by mreiche** [aurelia/dependency-injection#168]
I've recently updated to aurelia-dependency-injection 1.4 and I'm unable to bundle with JSPM:

```
jspm bundle myty/backend/services/* + myty/core/services/* + aurelia-framework + aurelia-bootstrapper + aurelia-loader-default + aurelia-templating-binding + aurelia-templating-router + aurelia-templating-resources + aurelia-pal + aurelia-pal-browser + aurelia-router + aurelia-history + aurelia-history-browser + aurelia-logging-console + aurelia-validation + aurelia-fetch-client + aurelia-plugins-cookies + aurelia-i18n + aurelia-animator-css + aurelia-loader + text + i18next-xhr-backend + url-parse + moment + aurelia-orm + aurelia-api ./tycon/modules/backend/js/myty-aurelia.min.js --skip-source-maps --minify 

err  Error on fetch for npm:aurelia-dependency-injection@1.4.0.js at file:///vendor/jspm_packages/npm/aurelia-dependency-injection@1.4.0.js
        Loading npm:aurelia-framework@1.2.0/aurelia-framework.js
        Loading npm:aurelia-framework@1.2.0.js
        Loading myty/backend/services/attributes.js
        Error: ENOENT: no such file or directory, open '/vendor/jspm_packages/npm/aurelia-dependency-injection@1.4.0.js
```

When I downgrade to `1.3.2` its everything fine, because `npm/aurelia-dependency-injection@1.3.2.js` is present.

**Comment by ben-girardet** [comment]
Same when using Webpack. See this post on Discourse: https://discourse.aurelia.io/t/try-to-install-aurelia-cli-but-get-some-issue/1351

**Comment by pete-hotchkiss** [comment]
this is all going to be - I suspect at least - related to #166 and #167

**Comment by EisenbergEffect** [comment]
Released 1.4.1. Please give that a try.

**Comment by mreiche** [comment]
`[16:54:15] Finished 'bundleProd' after 6.51 s`
